File: B/Java/Ram.java
// Kelas Ram
class Ram
{
    // Atribut Private
    private String capacity;
    private int price;

    // Constructor
    Ram()
    {
        
    }

    // Constructor Input
    public Ram(String capacity, int price)
    {
        this.capacity = capacity;
        this.price = price;
    }

    // Metode Getter dan Setter
    public String getCapacity()
    {
        return capacity;
    }

    public void setCapacity(String capacity)
    {
        this.capacity = capacity;
    }

    public int getPrice()
    {
        return price;
    }

    public void setPrice(int price)
    {
        this.price = price;
    }

    // Metode Print
    public void printRam()
    {
        System.out.println("==================================");
        System.out.println("Ram:");
        System.out.println("==================================");
        System.out.println("Kapasitas       : " + this.capacity);
        System.out.println("Harga           : " + "Rp" + this.price);
    }
}

File: B/Python/Ram.py
# Kelas Ram
class Ram:
    __capacity = ""
    __price = 0

    # Constructor
    def __init__(self, capacity ="", price = 0):
        self.__capacity = capacity
        self.__price = price

    # Metode Setter
    def setCapacity(self, capacity):
        self.__capacity = capacity

    def setPrice(self, price):
        self.__price = price

    # Metode Getter
    def getCapacity(self):
        return self.__capacity

    def getPrice(self):
        return self.__price
    
    # Metode Print
    def printRam(self):
        print("===============================")
        print("Ram:")
        print("===============================")
        print("Kapasitas    : " + self.getCapacity())
        print("Harga        : " + "Rp" + str(self.getPrice()))